deployment/stacks/parent.ts

```diff
@@ -45,16 +44,7 @@ export class ParentStack extends Stack {
     const vpc = ec2.Vpc.fromLookup(this, "Vpc", { vpcName });
     const kafkaAuthorizedSubnets = getSubnets(this, kafkaAuthorizedSubnetIds);
 
-    const loggingBucket = s3.Bucket.fromBucketName(
-      this,
-      "LoggingBucket",
-      `cms-cloud-${Aws.ACCOUNT_ID}-${Aws.REGION}`
-    );
```

Comment:
Typescript pointed out that this was not being used or required where it was being passed. Just wanted to make sure that isn't unexpected @JonHolman